src/includes/class-alma-wc-payment-gateway.php

```diff
@@ -171,6 +171,12 @@ public function process_admin_options() {
 	 * @override
 	 */
 	public function is_available() {
+
+		$current_user = wp_get_current_user();
+		if ( ! in_array( 'administrator', $current_user->roles, true ) && 'live' !== alma_wc_plugin()->settings->environment ) {
```

Comment:
sorry, I didn't shown that was added and not removed

you should try to reuse this method copied from alma_wc_plugin->run() instead of just copy the code (DRY)